packages/eslint-plugin/src/rules/no-unsafe-member-access.ts

```diff
@@ -107,8 +107,8 @@ export default createRule({
     }
 
     return {
-      // ignore MemberExpression if it's parent is TSClassImplements or TSInterfaceHeritage
-      ':not(TSClassImplements, TSInterfaceHeritage) > MemberExpression':
+      // ignore MemberExpression's with ancestor's of type `TSClassImplements` or `TSInterfaceHeritage`
```

Comment:
[Docs] Grammar nit: these are plurals, not possessives:

```suggestion
      // ignore MemberExpressions with ancestors of type `TSClassImplements` or `TSInterfaceHeritage`
```